### ISSUE TITLE
Navbar reloads every time a new page is opened 

### ISSUE THREAD

**Issue by NehaSan16** [frappe/wiki#235]
https://github.com/frappe/wiki/assets/139349833/e12432b5-5ebb-4f77-9a81-c2331711ece5

@BreadGenie I know you're not focussed on Wiki right now, but this is a buggy issue. Why do we reload and scroll the navbar every time a page opens? It leads to a poor reading experience. Can we please look into this?

CC: @rmehta

**Comment by BreadGenie** [comment]
> Why do we reload and scroll the navbar every time a page opens? It leads to a poor reading experience. Can we please look into this?

I agree with it being poor UX. This happens because Wiki is a static site. So everytime you navigate to a different page it'll load all the HTML again and again. We are doing it for a good reason which is SEO. This particular issue could be solved by either a Vue SSR rewrite or by adding client side routing using a router library (but I reckon it could impact SEO (take it with a pinch of salt though)). Both needs a good chunk of time to implement.

I'll take a look at this again once I make some time.

**Comment by NehaSan16** [comment]
Thanks for acknowledging. Let us implement the fix. 

Most of the traffic we get on the docs is directly from google search... so it might not impact SEO. 

Most other docs are significantly easy to navigate. eg. https://docs.stripe.com/payments

**Comment by NagariaHussain** [comment]
This was fixed by @pratikb64
